saleor/plugins/user_email/plugin.py

```diff
@@ -22,63 +29,61 @@
-    UserNotifyEvent.ORDER_PAYMENT_CONFIRMATION: send_payment_confirmation,
-    UserNotifyEvent.ORDER_CANCELED: send_order_canceled,
-    UserNotifyEvent.ORDER_REFUND_CONFIRMATION: send_order_refund,
-}
+def get_user_event_map():
+    return {
+        UserNotifyEvent.ACCOUNT_CONFIRMATION: send_account_confirmation,
+        UserNotifyEvent.ACCOUNT_SET_CUSTOMER_PASSWORD: (
+            send_account_set_customer_password
+        ),
+        UserNotifyEvent.ACCOUNT_DELETE: send_account_delete,
+        UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_CONFIRM: send_account_change_email_confirm,
+        UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_REQUEST: send_account_change_email_request,
+        UserNotifyEvent.ACCOUNT_PASSWORD_RESET: send_account_password_reset_event,
+        UserNotifyEvent.INVOICE_READY: send_invoice,
+        UserNotifyEvent.ORDER_CONFIRMATION: send_order_confirmation,
+        UserNotifyEvent.ORDER_FULFILLMENT_CONFIRMATION: send_fulfillment_confirmation,
+        UserNotifyEvent.ORDER_FULFILLMENT_UPDATE: send_fulfillment_update,
+        UserNotifyEvent.ORDER_PAYMENT_CONFIRMATION: send_payment_confirmation,
+        UserNotifyEvent.ORDER_CANCELED: send_order_canceled,
+        UserNotifyEvent.ORDER_REFUND_CONFIRMATION: send_order_refund,
+    }
 
 
 class UserEmailPlugin(BasePlugin):
     PLUGIN_ID = "mirumee.notifications.user_email"
     PLUGIN_NAME = "User emails"
     # TODO the configuration will be implemented in separate pull request
 
-    # DEFAULT_CONFIGURATION = [
-    #     {"name": "account-confirmation_template", "value": None},
-    #     {"name": "account-password-reset-template", "value": None},
-    # ]
-    #
-    # CONFIG_STRUCTURE = {
-    #     "account-confirmation_template": {
-    #         "type": ConfigurationTypeField.SECRET,
-    #         "help_text": "",
-    #         "label": "Account confirmation template",
-    #     },
-    #     "account-password-reset-template": {
-    #         "type": ConfigurationTypeField.SECRET,
-    #         "help_text": "",
-    #         "label": "Account password reset template",
-    #     },
-    # }
+    DEFAULT_CONFIGURATION = [] + DEFAULT_EMAIL_CONFIGURATION  # type: ignore
+    CONFIG_STRUCTURE = {}
+    CONFIG_STRUCTURE.update(DEFAULT_EMAIL_CONFIG_STRUCTURE)
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        # configuration = {item["name"]: item["value"] for item in self.configuration}
-        self.active = True
+        configuration = {item["name"]: item["value"] for item in self.configuration}
+        self.config = EmailConfig(
+            host=configuration["host"],
+            port=configuration["port"],
+            username=configuration["username"],
+            password=configuration["password"],
+            sender_name=configuration["sender_name"],
+            sender_address=configuration["sender_address"],
+            use_tls=configuration["use_tls"],
+            use_ssl=configuration["use_ssl"],
+        )
```

Comment:
Keys of configuration dict are exact match of kwarg names passed to `EmailConfig`, are there any other keys in configuration? if not, we can pass it as
`self.config = EmailConfig(**configuration)`?